firebase-app-distribution/src/main/java/com/google/firebase/appdistribution/FirebaseAppDistributionException.java

```diff
@@ -42,10 +42,6 @@
     /** Installation canceled */
     INSTALLATION_CANCELED,
 
-    /** No foreground activity available for a given intent */
-    // TODO(rachelprince): add this to API council review
-    FOREGROUND_ACTIVITY_NOT_AVAILABLE,
```

Comment:
Please remove this from from the equivalent file in `firebase-app-distribution-stub` as well, and update that api.txt file. At some point we'll add a submit check to make sure the API surfaces are the same.

Reply:
Good call! will do